app/src/main/java/io/gnosis/safe/ui/settings/app/NightModeAppSettingsFragment.kt

```diff
@@ -0,0 +1,81 @@
+package io.gnosis.safe.ui.settings.app
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.appcompat.app.AppCompatDelegate.*
+import androidx.navigation.fragment.findNavController
+import io.gnosis.safe.ScreenId
+import io.gnosis.safe.databinding.FragmentSettingsAppNightModeBinding
+import io.gnosis.safe.di.components.ViewComponent
+import io.gnosis.safe.ui.base.fragment.BaseViewBindingFragment
+import javax.inject.Inject
+
+class NightModeAppSettingsFragment : BaseViewBindingFragment<FragmentSettingsAppNightModeBinding>() {
+    override fun screenId() = ScreenId.SETTINGS_APP_ADVANCED
```

Comment:
please ask if we need to use separate screen id here and update https://github.com/gnosis/safe/wiki/Overview-screen-tracking if necessary